src/mount/sauna_client.h

```diff
@@ -332,6 +332,11 @@ struct RequestException : public std::exception {
 
 uint8_t get_session_flags();
 
+inline std::atomic<bool> isDisconnectedFromMaster;
```

Comment:
As it will be used on result from https://github.com/leil-io/windows-client/pull/7 where the clients has a new thread that request to that value at the same time it could be change during the client's main process.